build.fsx

```diff
@@ -107,6 +107,21 @@ Target "CreatePackageEventFlowReadStoresMsSql" (fun _ ->
             "Source/EventFlow.ReadStores.MsSql/EventFlow.ReadStores.MsSql.nuspec"
     )
 
+Target "CreatePackageEventFlowReadStoresElasticsearch" (fun _ ->
+    let binDir = "Source/EventFlow.ReadStores.Elasticsearch/bin/"
+    CopyFile binDir (binDir + buildMode + "/EventFlow.ReadStores.Elasticsearch.dll")
+    NuGet (fun p ->
+        {p with
+            OutputPath = dirPackages
+            WorkingDir = "Source/EventFlow.ReadStores.Elasticsearch"
+            Version = nugetVersion
+            Dependencies = [
+                "EventFlow",  nugetVersionDep
+                "EventFlow.Elasticsearch",  nugetVersionDep]
```

Comment:
@tabro Delete this line as we don't have this NuGet package, or will you create that as well?